meta/vote-text-checklist.md

```diff
@@ -0,0 +1,12 @@
+
+# Rocket Pool Vote Text Checklist
+
+- [ ] Vote text should include the following:
+  - [ ] A short one-line summary intended to display effectively on the snapshot vote-listing page.
+  - [ ] Context as to the reason for the vote.
+  - [ ] A clear and concrete explanation of the possible outcomes of the voting process.
+  - [ ] Links to relevant resources or further context.
+  - [ ] A link to a relevant discussion thread on the Rocket Pool forums.
+- [ ] Ensure that all URLs point to the intended destinations.
+- [ ] The issue of framing bias has been considered and mitigated where possible.
```

Comment:
Might be simpler to say something like `Ensure the vote is presented as neutrally as possible.`